src/main/resources/assets/advancedperipherals/lang/zh_cn.json

```diff
@@ -54,7 +54,7 @@
   "item.advancedperipherals.tooltip.me_bridge": "&7ME网关可以与您的Applied Energistics系统交互",
   "item.advancedperipherals.tooltip.memory_card": "&7可以保存并给予库存管理器玩家背包的交互权限。",
   "item.advancedperipherals.tooltip.memory_card.bound": "&7已绑定到 &b%s&7.",
-  "item.advancedperipherals.tooltip.nbt_storage": "&7NBT版存储硬盘, 可以存储基于NBT的数据",
+  "item.advancedperipherals.tooltip.nbt_storage": "§7NBT版存储硬盘, 可以存储基于NBT的数据",
```

Comment:
Are you sure it should be `§`? All of the others are using `&` instead of `§`

Reply:
Yes it should be & and not §. Not sure who made that change, but I am going to revert that before the next release

Reply:
Thanks for noticing

Reply:
Not sure why that change came from me, probably when I uploaded some backups from the time the GitHub integration broke.
I accept your translation